docs/application/native/guides/device/settings.md

```diff
@@ -177,8 +177,10 @@ The following table lists the available system setting keys.
 | `SYSTEM_SETTINGS_KEY_3G_DATA_NETWORK_ENABLED` | `bool`   | `GETTER, NOTIFIER`      | Indicates whether the 3G data network is enabled. |
 | `SYSTEM_SETTINGS_KEY_ACCESSIBILITY_TTS`  | `bool`   | `GETTER, NOTIFIER`      | Indicates whether the accessibility TTS is enabled on the device. |
 | `SYSTEM_SETTINGS_KEY_ADS_ID`             | `string` | `GETTER, NOTIFIER`      | Ads ID for each device.                  |
```

Comment:
Is it the word ADD or Advertisements contraction (AD)?
Then change spelling to "Adds ID for each device."

Reply:
it's Advertisements contraction.
I fixed to "ADS ID".

Reply:
Thanks for clarifying, for clarity recommend changing it to "Advertisement ID for each device."